migrate_pim/apply_patch/apply_patch_ee_flexibility.rst

```diff
@@ -0,0 +1,73 @@
+How to apply a patch - Enterprise Edition - Flexibility Cloud offer
+========================================================================
+
+In the following example, Akeneo PIM version 2.3.10 has just been released and we are using an Akeneo PIM version 2.3.9.
+
+We always tag both Community and Enterprise versions with aligned version numbers, be sure to use the exact same version for CE and EE, for instance, a EE 2.3.10 fix may depend on CE 2.3.10.
+
+Using the exact patch version will avoid any local composer cache issue.
+
+In your enterprise standard project, the composer.json will reference both community and enterprise bundles repositories.
+
+To upgrade, please change the composer.json to:
+
+.. code-block:: javascript
+
+    {
+        ...
+        "akeneo/pim-enterprise-dev": "2.3.10",
+        "akeneo/pim-community-dev": "2.3.10",
+        ...
+    }
+
+.. note::
+
+    Before launching the following commands, remember to stop the daemon to avoid generating outdated cache.
```

Comment:
The client/partner does not have the hand on the daemon, so I don't know what we should write here.. :(

Reply:
I remove this part and add it at the end :

After that you can relaunch fpm to avoid generating outdated cache:

.. code-block:: bash
    partners_php7.1-fpm restart